src/ec/suite_b/ops/p256.rs

```diff
@@ -270,32 +270,56 @@ fn p256_scalar_inv_to_mont(a: &Scalar) -> ScalarMont {
 
 
 extern {
-    fn GFp_nistz256_add(r: *mut Limb/*[COMMON_OPS.num_limbs]*/,
-                        a: *const Limb/*[COMMON_OPS.num_limbs]*/,
-                        b: *const Limb/*[COMMON_OPS.num_limbs]*/);
-    fn GFp_nistz256_mul_mont(r: *mut Limb/*[COMMON_OPS.num_limbs]*/,
-                             a: *const Limb/*[COMMON_OPS.num_limbs]*/,
-                             b: *const Limb/*[COMMON_OPS.num_limbs]*/);
-    fn GFp_nistz256_sqr_mont(r: *mut Limb/*[COMMON_OPS.num_limbs]*/,
-                             a: *const Limb/*[COMMON_OPS.num_limbs]*/);
-
-    fn GFp_nistz256_point_add(r: *mut Limb/*[3][COMMON_OPS.num_limbs]*/,
-                              a: *const Limb/*[3][COMMON_OPS.num_limbs]*/,
-                              b: *const Limb/*[3][COMMON_OPS.num_limbs]*/);
-    fn GFp_nistz256_point_mul(r: *mut Limb/*[3][COMMON_OPS.num_limbs]*/,
-                              p_scalar: *const Limb/*[COMMON_OPS.num_limbs]*/,
-                              p_x: *const Limb/*[COMMON_OPS.num_limbs]*/,
-                              p_y: *const Limb/*[COMMON_OPS.num_limbs]*/);
-    fn GFp_nistz256_point_mul_base(r: *mut Limb/*[3][COMMON_OPS.num_limbs]*/,
-                                   g_scalar: *const Limb/*[COMMON_OPS.num_limbs]*/);
-
-    fn GFp_p256_scalar_mul_mont(r: *mut Limb/*[COMMON_OPS.num_limbs]*/,
-                                a: *const Limb/*[COMMON_OPS.num_limbs]*/,
-                                b: *const Limb/*[COMMON_OPS.num_limbs]*/);
-    fn GFp_p256_scalar_sqr_mont(r: *mut Limb/*[COMMON_OPS.num_limbs]*/,
-                                a: *const Limb/*[COMMON_OPS.num_limbs]*/);
-    fn GFp_p256_scalar_sqr_rep_mont(r: *mut Limb/*[COMMON_OPS.num_limbs]*/,
-                                    a: *const Limb/*[COMMON_OPS.num_limbs]*/,
+    fn GFp_nistz256_add(// [COMMON_OPS.num_limbs]
```

Comment:
Had to make these `//` comments so that it wouldn't try to put them on the same line, and then fall afoul of the line length limit